### ISSUE TITLE
Invite system is sorta broken, error in parsing JSON from RestAction.

### ISSUE THREAD

**Issue by BlueMustache** [DV8FromTheWorld/JDA#274]
I'm running JDA-3.0.BETA2_142_withDependencies and I've seen this issue persist at least from _140.
The Invite system is successful, but spits out a message about a parsing error of the attribute max_age. This may or may not be deprecated in the Discord docs. I would appreciate a quick patch for this, as it's quite annoying. Here is the stacktrace. Thanks and keep up the great work!

```
[22:18:30] [Fatal] [JDARequester]: Requester system encountered an internal error
[22:18:30] [Fatal] [JDARequester]: Encountered an exception:
[22:18:30] [Fatal] [JDARequester]: org.json.JSONException: JSONObject["max_uses"] not found.
	at org.json.JSONObject.get(JSONObject.java:471)
	at org.json.JSONObject.getInt(JSONObject.java:598)
	at net.dv8tion.jda.core.entities.EntityBuilder.createInvite(EntityBuilder.java:1170)
	at net.dv8tion.jda.client.entities.impl.JDAClientImpl$1.handleResponse(JDAClientImpl.java:222)
	at net.dv8tion.jda.core.requests.Requester.execute(Requester.java:150)
	at net.dv8tion.jda.core.requests.ratelimit.ClientRateLimiter$Bucket.run(ClientRateLimiter.java:200)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)

[22:18:30] [Fatal] [RestAction]: RestAction queue returned failure: [JSONException] JSONObject["max_uses"] not found.
```

**Comment by MinnDevelopment** [comment]
We have already spotted this and it is on our internal tracker, thank you for reporting it.

**Comment by BlueMustache** [comment]
Awesome. Thanks. Impressive response time as well.

**Comment by DV8FromTheWorld** [comment]
This has been fixed as of 3b33384095a6ef98da5c2e6280f39480ef47e84f
